frontend/src/Leaderboard/index.jsx

```diff
@@ -20,9 +22,29 @@ const mockedUsers = [
   },
 ];
 
+const styles = StyleSheet.create({
+  searchBar: {
+    width: '100%',
+    borderRadius: 10,
+    backgroundColor: Colors.lightenGrey,
+  },
+  searchBarView: {
+    flexDirection: 'row',
+    margin: 2,
+  },
+});
+
 function Leaderboard() {
+  function mockedSearchUser(username) {
+    // eslint-disable-next-line no-alert
+    alert(`search user ${username}`);
+  }
```

Comment:
why do you put an alert here? just for testing?

Reply:
Yes, just to show the submit function works.